source/snippets/install_notes.rst_

```diff
@@ -0,0 +1,10 @@
+.. note::
```

Comment:
Shall we also include this snippet on the landing page, since
not everyone will read the template.

Reply:
I meant to include the template at the top of **all** installation sections.

Reply:
But the template may have something unrelated to the special software. e.g., we don't need git to install fk.

Reply:
Yes, that's why I say it's a general note.

Reply:
Perhaps, now it is the best way to just put the snippet at the top of all installation sections.

Reply:
Some C packages don't need gfotran, but as gfortran is very common, it's always good to have it installed. 

These 5 commands are probably the most commonly used ones.

Reply:
I agree~